packages/main/src/RPA/Excel/Files.py

```diff
@@ -857,7 +919,14 @@ def set_cell_format(self, row, column, fmt, name=None):
         assert self.workbook, "No active workbook"
         self.workbook.set_cell_format(row, column, fmt, name)
 
-    def insert_image_to_worksheet(self, row, column, path, scale=1.0, name=None):
+    def insert_image_to_worksheet(
+        self,
+        row: int,
+        column: Union[int, str],
+        path: str,
+        scale: Optional[float] = 1.0,
```

Comment:
```suggestion
        scale: float = 1.0,
```
scale should be a `float` (or number) at all times.